pkgs/development/libraries/libevent/default.nix

```diff
@@ -1,4 +1,9 @@
-{ stdenv, fetchurl, openssl, findutils }:
+{
+  stdenv, fetchurl, findutils,
+  enableOpenssl? true, openssl
```

Comment:
It seems that `sslSupport` is more commonly used in Nixpkgs. So I would suggest to use `sslSupport ? true` here.